### DISCUSSION TITLE
Server Error Question

### DISCUSSION THREAD

**Discussion by Aarons-Joshua** [cooperrc/computational-mechanics#42] [Q&A]
Does anyone know how to resolve this problem: 

There was a server error on jupyter and told me to sign out. Then, once I signed back in, all my work was erased and I don't know if it is gone for good. If anyone knows how to fix this problem please let me know. I really hope all my work is not lost :(

**Comment by y1bo44** [comment]
it happen to me the same and i think there is something wrong with the server.I tried to log out for several times and it showed nothing.

**Comment by VictoireHA** [comment]
My work is lost:(

**Comment by FishMacaron** [comment]
Yeah Jupyter notebook tends to be like that from my experience (crashes a lot from server errors). I'm not sure how to fix it unfortunately but I recommend copying and pasting your code somewhere you know it can't be lost like a google doc or some sort of notes app just in case as a backup. That's what I did so I was lucky before mine crashed earlier today.
(I know this is a repost of my comment earlier but it's bothering me that I accidentally made it a reply instead of its own comment :,) )

**Comment by Aidan-Myatt** [comment]
What works for me when this happens is to keep the notebook you're working on with all your code in it open. Then, just open another tab and try to log into the server as normal, and it will ask if want to proceed. Click yes and then go back to the open notebook with your work and click save. That is what I have been doing, and I have not lost my work.

**Comment by cooperrc** [comment]
**Always​ save your work in the work​ folder.**

If you open the file from the rocket launch button, click the big Jupyter icon in the upper left. and select the "computational-mechanics" folder:
![image](https://user-images.githubusercontent.com/11371428/172878976-be7cdea3-1ad4-454a-8404-5113b8a49f9a.png)

Then, choose "Move" and type: /work/​
![image](https://user-images.githubusercontent.com/11371428/172879024-c265487e-dbb7-4098-bad9-67e199d333f4.png)



Now, your work is in a persistent folder that will not be erased.
